tests/end-to-end/external-solution-integration/Dockerfile

```diff
@@ -12,6 +12,7 @@ LABEL source=git@github.com:kyma-project/kyma.git
 
 COPY --from=builder /e2e /
 
-RUN apk update && apk add ca-certificates && apk add curl && rm -rf /var/cache/apk/*
+# temporarily install latest cURL to mitigate the security issues
+RUN apk update && apk add ca-certificates && apk add --no-cache --update-cache --repository http://nl.alpinelinux.org/alpine/edge/main/ curl && rm -rf /var/cache/apk/*
```

Comment:
As we updating the alpine cache to `edge` I think 
```
FROM alpine:3.10
```
this is a smaller change that will do that job.

At the same time, let's create an issue on us to change `edge` later to a released version when it is possible.

Reply:
@sayanh I can upgrade the `apline` version to the latest, however, I tested that locally and the `curl` version available is still not the latest `curl-7.69`